src/app/controllers/admin-forms.server.controller.js

```diff
@@ -218,88 +49,6 @@ function makeModule(connection) {
       }
       return next()
     },
-    /**
-     * Updates a form from Settings tab
-     * @param  {Object} req - Express request object
-     * @param  {Object} res - Express response object
-     */
-    update: function (req, res) {
-      // form is retrieved from formById and updatedForm is passed to us from frontend
-      let form = req.form
-      let updatedForm = req.body.form
-
-      // Admin from frontend has to be deleted because it contains populated agency
-      delete updatedForm.admin
-      delete updatedForm.__v
-      delete updatedForm._id
-      delete updatedForm.id
-      delete updatedForm.created
-      delete updatedForm.lastModified
-      delete form.__v
-      delete form.created
-      delete form.lastModified
-
-      if (!_.isEmpty(updatedForm.editFormField)) {
-        if (!_.isEmpty(updatedForm.form_fields)) {
-          // form_fields should not exist in updatedForm
-          logger.error({
-            message: 'form_fields should not exist in updatedForm',
-            meta: {
-              action: 'makeModule.update',
-              ...createReqMeta(req),
-              formId: form._id,
-            },
-          })
-          return res
-            .status(StatusCodes.BAD_REQUEST)
-            .json({ message: 'Invalid update to form' })
-        } else {
-          const { error, formFields } = getEditedFormFields(
-            _.cloneDeep(form.form_fields),
-            updatedForm.editFormField,
-          )
-          if (error) {
-            logger.error({
-              message: 'Error getting edited form fields',
-              meta: {
-                action: 'makeModule.update',
-                ...createReqMeta(req),
-                formId: form._id,
-              },
-              error,
-            })
-            return res.status(StatusCodes.BAD_REQUEST).json({ message: error })
-          }
-          form.form_fields = formFields
-          delete updatedForm.editFormField
-        }
-      }
-
-      // Log updates to customLogo so that we know when admin clients are no longer updating customLogo
-      if (
-        updatedForm.customLogo !== undefined &&
-        form.customLogo !== updatedForm.customLogo
-      ) {
-        logger.info({
-          message: `Custom logo being updated for form ${form._id}`,
-          meta: {
-            action: 'makeModule.update',
-          },
-        })
-      }
-
-      _.extend(form, updatedForm)
-
-      // Updates to logo should also update customLogo (to account for clients still referencing customLogo)
-      // TODO: Remove once all forms have logo key and customLogo is removed from schema
-      updateCustomLogoInForm(form)
```

Comment:
is it safe to delete this line? we still have the `logo` key in `startPage` right?

Reply:
Was following https://github.com/opengovsg/FormSG/issues/787, but I guess I'll keep it in and let whoever is in charge of that issue clear it later on

Reply:
have we done the "Verify that it is safe to remove the customLogo key on the Form schema" part?

Reply:
Nope, i'll add it back in

Reply:
Actually, let me query the update log, if it's not been triggered in the past few months I'll remove it.

Reply:
even if it hasn't appeared, I'm not sure it's safe to remove this. the form model still has `startPage.logo`, `logo` as well as `customLogo`. I'm not sure which one is correct and whether we need to do a db migration before we remove this

Reply:
@liangyuanruo will require your input here on whether to add back in the check for customLogo changes. 

The check removed in this refactor since Joi validation already prevents customLogo from being passed as a key-value, but I can add that back in

Reply:
fwiw, there has been no updates to customLogo for the past 4 months:
![Screenshot 2020-12-28 at 12 11 35 PM](https://user-images.githubusercontent.com/22133008/103189045-e4f72b80-4905-11eb-8f50-664c5fd8b8e2.png)

Reply:
Honestly, I don't recall - will need @arshadali172 to advise

Reply:
addressed in https://github.com/opengovsg/FormSG/pull/979